tai-assignments.md

```diff
@@ -0,0 +1,20 @@
+# Trust Anchor Identifier Assignments
+The following table tracks the list of OIDs assigned as Trust Anchor Identifiers.
+
+## Steps to assign new TAIs:
+1. Obtain a Private Enterprise Number OID from IANA ([Request Form](https://www.iana.org/assignments/enterprise-numbers/assignment/apply/))
+2. For each participating trust anchor, identified by their (subjectName, public key) tuple, assign a unique OID under the PEN.
+3. Submit a pull request that adds a new row to the below table for each TAI assignment.
+
+### Notes:
+* TAI OIDs should be represented in dot decimal notation.
+* Public Keys should be PEM-encoded and include the `-----BEGIN PUBLIC KEY-----` and `-----END PUBLIC KEY-----` blocks. 
+* The following command can be used to easily extract public keys in the correct format:
+    * `$ openssl x509 -in certificate.pem -noout -pubkey`
+* The following command can be used to easily extract subjectNames in the correct format:
+    * `$ openssl x509 -in certificate.pem -noout -subject | sed -e "s/^subject= //"`
```

Comment:
Oops, sorry missed this. LGTM but one comment: this uses the text form of the subject name, I think close-ish to RFC 2253 though OpenSSL's API has lots of options and I don't know which they use for `-subject`.

The problem is RFC 2253's syntax is not actually enough to give you back the name. It might be enough up to normalization, but normalization is kind of cursed. (The text syntax doesn't tell you what string type each field used.) I'm not sure what's the best thing to do here. There isn't an easy way to get a hex dump of the subject name.